caffe2/python/cnn.py

```diff
@@ -521,6 +521,22 @@ def Iter(self, blob_out, **kwargs):
             **kwargs)
         return self.net.Iter(blob_out, blob_out, **kwargs)
 
+    def Accuracy(self, blob_in, blob_out, **kwargs):
```

Comment:
I can do this, but I won't be able to get to it for a little bit.

@pooyadavoodi it should be done in this PR, no sense in adding functionality and then immediately moving it.